payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/DropdownFieldUI.kt

```diff
@@ -96,32 +95,46 @@ internal fun DropDown(
                     expanded = true
                 }
         ) {
-            Column(
-                modifier = Modifier.padding(
-                    start = 16.dp,
-                    top = 4.dp,
-                    bottom = 8.dp
-                )
-            ) {
-                label?.let {
-                    FormLabel(stringResource(it), enabled)
-                }
-                Row(
-                    modifier = Modifier.fillMaxWidth(),
-                    verticalAlignment = Alignment.Bottom
-                ) {
+            if (controller.tinyMode) {
```

Comment:
It feels like tinyMode is kinda like wrap content width, and no label?  It feels better to me to have each of these properties exposed in the interface separately.

I am curious on your thoughts on if these properties should be in the controller or just the in UI as is the case for enabled?  I am not really sure myself and thinking through it some more.

Reply:
I think this is better, the Compose method knows how to render `tinyMode` or regular. There are other details like the Row verticalAlignment which is different for each mode. We'll be putting information about how to render into the controller.